demo/src/main/res/layout/item_view_rateable_movie.xml

```diff
@@ -43,8 +43,9 @@
         android:layout_marginEnd="8dp"
         android:layout_marginStart="8dp"
         android:layout_marginTop="8dp"
-        android:stepSize="0.5"
+        android:importantForAccessibility="no"
```

Comment:
could/should also apply this to the `like` imageview. would just need to check on other API versions to verify that it's enough - that way we don't need to unbind clicklisteners/do `view.setClickable(false)` when we're doing the indirect access branch.